dev/Installer/src/install.h

```diff
@@ -9,8 +9,8 @@ namespace ProjectReunionInstaller {
         wil::unique_cotaskmem_string fullName;
         wil::unique_cotaskmem_string familyName;
         winrt::Windows::System::ProcessorArchitecture architecture = winrt::Windows::System::ProcessorArchitecture::Unknown;
-        std::uint64_t version = 0u;
+        std::uint64_t version{ 0u };
```

Comment:
NIT: `std::uint64_t version{};`

No need to explicitly define the value as {} gets you the default for the type, which is 0 for integers.